theatre/studio/src/panels/SequenceEditorPanel/RightOverlay/FocusRangeZone/FocusRangeThumb.tsx

```diff
@@ -37,6 +42,28 @@ const HitZone = styled.div<{enabled: boolean; type: 'start' | 'end'}>`
     pointer-events: ${(props) => (props.enabled ? 'auto' : 'none')};
   }
 
+  #pointer-root.draggingPositionInSequenceEditor & {
+    pointer-events: auto;
+    cursor: none;
+
+    &:hover:after {
+      position: absolute;
+      top: calc(50% - ${snapCursorSize / 2}px);
+      left: calc(50% - ${snapCursorSize / 2}px);
+      width: ${snapCursorSize}px;
+      height: ${snapCursorSize}px;
+      display: block;
+      content: ' ';
+      background: url(${SnapCursor}) no-repeat;
+      background-size: cover;
+      z-index: 30;
+    }
+  }
+
+  &.dragging {
+    pointer-events: none !important;
+  }
+
```

Comment:
Before:

https://user-images.githubusercontent.com/593118/166229005-602c0a38-1e25-4dfd-a744-59ea29398a56.mp4

After:


https://user-images.githubusercontent.com/593118/166229020-d8577f44-bdd8-47f1-8f28-92763f5f486f.mp4

(Visuals are tweaked in a later commit)